acpdr/swagger-specs/audit-query.yaml

```diff
@@ -0,0 +1,332 @@
+      operationId: exportUsingGET
+      produces:
+      - application/octet-stream
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-request-id'
+        - $ref: '#/parameters/property'
+      responses:
+        '204':
+          description: A successful response that doesn't generate any results returns HTTP status 204 (No Content).
+        '307':
+          description: A successful response that generates results returns HTTP status 307 (Temporary Redirect). A link to the exported CSV file is provided in the `Location` response header.
+        '400':
+          description: The request was malformed. Ensure that you are including all required headers before trying again.
+        '404':
+          description: The resource was not found.
+parameters:
+  authorization:
+    name: Authorization
+    description: 'The access token which can be copied from your Experience Platform integration, prefixed with "Bearer ". 
+    For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).'
+    required: true
+    type: string
+    in: header
+  x-gw-ims-org-id:
+    name: x-gw-ims-org-id
+    description: 'The Organization ID which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).'
+    required: true
+    type: string
+    in: header
+  x-api-key:
+    name: x-api-key
+    description: 'The API key which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).'
+    required: true
+    type: string
+    in: header
+  x-sandbox-name:
+    name: x-sandbox-name
+    description: 'The name of the sandbox in which the operation will take place. See the [sandboxes overview documentation](https://adobe.com/go/sandbox-overview-en) for more information.'
+    required: true
+    type: string
+    in: header
+  property:
+    name: property
+    in: query
+    description: >-
+      You can filter list results by including a `property` query parameter. The value must be an array that contains one or more of a comma-separated list of properties. For example: `?property=action==create,assetType==Sandbox`.
+      
+      
+      The properties that can be used for filtering are `timestamp`, `status`, `action`, `user`, and `assetType`.
+    required: false
+    type: array
+    items:
+      type: string
+    collectionFormat: multi
+  x-request-id:
+    in: header
+    name: x-request-id
+    type: string
+    description: A tracing ID associated with the request. If one is not provided
```

Comment:
Let's keep the description on the same line, since I don't think it needs the newline here.